tests/src/Functional/DeveloperAppAnalyticsTest.php

```diff
@@ -228,4 +248,36 @@ protected function exportAnalyticsTest() {
     $this->assertEquals(403, $this->getSession()->getStatusCode());
   }
 
+  /**
+   * Add an app analytics mock response to the stack.
+   */
+  protected function queueAppAnalyticsStackedResponse() {
+    $this->stack->queueMockResponse([
+      'app_analytics' => [],
+    ]);
+  }
+
+  /**
+   * Add an app analytics mock response to the stack.
+   *
+   * @param \Drupal\apigee_edge\Entity\DeveloperAppInterface $app
+   *   The app.
+   * @param int $response_code
+   *   Response code, defaults to 200.
+   */
+  protected function queueDeveloperAppResponse(DeveloperAppInterface $app, $response_code = 200) {
```

Comment:
Should this function be in `ApigeeMockApiClientHelperTrait` instead for other tests to use?

Reply:
@arlina-espinoza Let's move it to the trait. I have a few tests that could use this.